src/django/oar/settings.py

```diff
@@ -11,6 +11,9 @@
 """
 
 import os
+import requests
+
+from django.core.exceptions import ImproperlyConfigured
```

Comment:
Should be able to get this import via a rebase of `develop`.

Reply:
I did a quick rebase but wasn't able to absolve these changes.